.github/workflows/acceptance.yml

```diff
@@ -0,0 +1,37 @@
+name: BrowserStack Acceptance Tests
+
+on: push
+
+jobs:
+  browserstack-acceptance-tests:
+    name: BrowserStack Acceptance Tests
+    runs-on: ubuntu-latest
+    env:
+      BROWSERSTACK_USERNAME: ${{ secrets.BROWSERSTACK_USERNAME }}
+      BROWSERSTACK_ACCESS_KEY: ${{ secrets.BROWSERSTACK_ACCESS_KEY }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 14
+      - run: npm ci
+      - run: npm run setup-test-site
+      - run: npm run build-test-site
+      - run: chmod +x ./.github/run_browserstack_acceptance.sh
+      - name: Run Acceptance Tests
+        run: ./.github/run_browserstack_acceptance.sh
+        shell: bash
+  headless-acceptance-tests:
+    name: Headless Acceptance Tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 14
+      - run: npm ci
+      - run: npm run setup-test-site
+      - run: npm run build-test-site
+      - name: Run Acceptance Tests
+        run: node tests/acceptance/run-tests.js --browsers chrome:headless safari:headless
```

Comment:
safari:headless isn't a testcafe browser yet, though we could probably download our own if we really wanted (or just use browserstack)

Reply:
Updated so that sfari is in browserstack and firefox is headless, and the tests passed